tools/README.md

```diff
@@ -167,3 +167,70 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Change edge type to canonical edge type for partitioned graph data
+
+`change_etype_to_canonical_etype.py` is a tool that helps you convert etypes to canonical_etypes in a partition configuration file, and it overwrites the original file on disk. 
+
+### Sample Usage
+
+```
+python tools/change_etype_to_canonical_etype.py --part-config "{config path}"
+```
+
+### Requirement
+
+Partition algorithms produce one configuration file and multiple data folders, and each data folder corresponds to a partition. **This tool requires not only access to the specified config file in input, but also to the data of the first partition.** They can be local files or shared files among network, if you follow this [official tutorial](https://docs.dgl.ai/en/latest/tutorials/dist/1_node_classification.html#sphx-glr-tutorials-dist-1-node-classification-py) for distributed training, you don't need to care about this as all files are shared by every partiticate through NFS.
```

Comment:
`every partiticate` incorroect word, shoud be a noun.

Reply:
`but also the data`

Reply:
how about say more about what need to be read? graph data of first partition? `graph.dgl`?

Reply:
`the first partition` seems not accurate, as we may need to access all partitions though it's unlikely?

Reply:
> `every partiticate` incorroect word, shoud be a noun.

use participant instead.

Reply:
> `the first partition` seems not accurate, as we may need to access all partitions though it's unlikely?

In most of the ceases, only part0 is needed. It has been noted as potential risk in the design doc, and we give user prompt and hint when risk raised. Here we don't mention this because it may make user confused.

Reply:
> how about say more about what need to be read? graph data of first partition? `graph.dgl`?

Make sense, it is more accurate.